### ISSUE TITLE
3 Extensions

### ISSUE THREAD

**Issue by rbtsolis** [vscode-django/vscode-django#14]
I think we have 3 extensions of django in the marketplace, but i think its better if we work in one :/, i have one, Scott have one, and you have one too, what do you think?

**Comment by batisteo** [comment]
Hi Roberth, thanks for reaching!
I’m aware of this as I’m the author of the (for now) last Django extension, so I made the difficult choice to create a new project instead of contributing. Thing that I usually do a lot. But now I didn’t wanted to wait for approval or discuss snippet style.

This extension has several goals:
- Learning about JavaScript and TypeScript
- Having the best syntax for templates, HTML and other (txt for emails…)
- Functionality par with PyCharm Pro
- Using the most of what VS Code can offer

So far I’ve got:
- A template syntax which is much more closer to what my perfectionist heart desire
- Some template snippets that can paste what you had previously copied in your clipboard
- Click-through (_Go To Definition_) on template paths in the templates, absolute or relative
- Python snippets that shows up on the right files (no Form fields snippets in the models.py)
- Python snippet(s) with case transformation (MyModel => verbose_name = _("my_model"))
- Configuration settings to disable some of the Python snippets.

Yet there’s still plenty to do until I burn out (Cf issues).

Obviously there some kind of hope that other contributors would join effort on mine — yeah, that’s kind of egoist — and that’s why I created this repository not under my pseudo but a more generic name. Having diversity isn’t that bad _per se_ as it allow exploring different ways. But choice could be irritating for users, that's for sure.

I didn’t really answered the question, but just gave a bit of context and _why’s_.

What do you think? Are you waiting first for an answer in https://github.com/ScottBarkman/vscode-djaneiro/issues/16 ?

**Comment by dangayle** [comment]
The biggest/first Django extension is not good. It doesn't parse HTML properly and the developer doesn't have Github issues turned on. This is the first good one I've found.

**Comment by rbtsolis** [comment]
Yes, thank you, i understand that, cool, too many reasons. i will try the extension, and work on it
